device-types/Cisco/WS-C4500X-16SFP+.yaml

```diff
@@ -49,6 +49,6 @@ module-bays:
   - name: Network Module
     position: '2'
   - name: PS1
-    position: '1'
```

Comment:
The issue with changing the module bay position, is that the module variable in the module template will only be expecting a number rather than PS1.